{
    "body": "### Steps to reproduce the issue\r\n```console\r\n$ spack install py-grpcio\r\n```\r\n\r\n### Information on your system\r\n\r\n* **Spack:** 0.17.1-1070-5208fcf\r\n* **Python:** 3.8.12\r\n* **Platform:** linux-rhel6-broadwell\r\n* **Concretizer:** clingo\r\n\r\n\r\n### Additional information\r\n\r\n[spack-build-env.txt](https://github.com/spack/spack/files/8003480/spack-build-env.txt)\r\n[spack-build-out.txt](https://github.com/spack/spack/files/8003481/spack-build-out.txt)\r\n\r\n@adamjstewart  \r\n@aweits\r\n\r\n\r\n### General information\r\n\r\n- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform\r\n- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers\r\n- [X] I have uploaded the build log and environment files\r\n- [X] I have searched the issues of this repo and believe this is not a duplicate",
    "user": "nkgh77",
    "url": "https://api.github.com/repos/spack/spack/issues/28781",
    "updated_at": "2022-02-06 09:33:58",
    "created_at": "2022-02-04 13:48:48",
    "closed_at": "2022-02-06 09:33:57",
    "state": "closed",
    "title": "Installation issue: py-grpcio",
    "number": 28781,
    "milestone": null,
    "labels": [
        "build-error"
    ],
    "id": 1124224208,
    "html_url": "https://github.com/spack/spack/issues/28781",
    "assignees": [],
    "comments": 2
}